src/Model/BarcodeScanResult.php

```diff
@@ -0,0 +1,158 @@
+<?php
+declare(strict_types=1);
+
+namespace DexproSolutionsGmbh\SoftekBarcodeWrapper\Model;
+
+
+class BarcodeScanResult
```

Comment:
This class either needs a constructor initializing all fields that have no default values or those fields must have default values. In Squeeze this often was the error for serialization errors.

Reply:
Yeah, i can understand what you mean but in our context an uninitialized barcode should not exist. Default Value can interpreted differently from users i would say that the integrating dev should create his own instance. 

This library will permanantly return consitent data.

Reply:
Okay, let's see how this will behave. We can always change it :+1: